### PR TITLE
Laravel Livewire documentation extension

### PR DESCRIPTION
**Pull request by tafhyseni**
## Description

Adds Laravel Livewire Documentation to Raycast

- New extension

## Screencast

Included within extension's readme
## Checklist

- [ Done ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ Done ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @tafhyseni 👋 

Awesome extension 🔥 – I have a few ideas, suggestions, and corrections for you to work on:

- Could be awesome if the icon had transparent background
- Could be cool to add versions like it's done in #607 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by tafhyseni** [comment]
Hello @pernielsentikaer 🤙

Thank you so much!
I have already made two changes:

- - Added versions via json files which get pulled by selected preference
- - Replaced current icon with a new icon without transparent

Please, let me know if you propose further changes and/or addons!

File: extensions/laravel-livewire/package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      "description": "Search Livewire",
+      "mode": "view",
+      "preferences": [
+        {
+          "name": "livewireVersion",
+          "type": "dropdown",
+          "required": true,
+          "title": "Livewire Version",
+          "description": "Specify the Laravel livewire version it will search the docs for.",
+          "default": "1.x",
```
Comment (pernielsentikaer) [inline extensions/laravel-livewire/package.json:23]:
```suggestion
          "default": "2.x",
```
Wouldn't it make more sense to have the newest as default?

File: extensions/laravel-livewire/src/documentation/1.x.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+  "Start": [
+    { "url": "https://laravel-livewire.com/docs/1.x/quickstart", "title": "Quickstart1" },
```
Comment (pernielsentikaer) [inline extensions/laravel-livewire/src/documentation/1.x.json:3]:
```suggestion
    { "url": "https://laravel-livewire.com/docs/1.x/quickstart", "title": "Quickstart" },
```
Remove debug number 😄

File: extensions/laravel-livewire/src/documentation/2.x.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{
+  "Start": [
+    { "url": "https://laravel-livewire.com/docs/2.x/quickstart", "title": "Quickstart2" },
```
Comment (pernielsentikaer) [inline extensions/laravel-livewire/src/documentation/2.x.json:3]:
```suggestion
    { "url": "https://laravel-livewire.com/docs/2.x/quickstart", "title": "Quickstart" },
```
Remove debug number 😄

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Wow, @tafhyseni! 🚀 

It looks so gorgeous with the new corrections, MUCH better!

I left a few comments for you to work with before we're ready to merge 💎 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by tafhyseni** [comment]
Hey @pernielsentikaer 

I think we are READY 🤩

**Review by pernielsentikaer — APPROVED** [review]
Yeah, @tafhyseni! 🔥 

I also think we're ready 🚀  I'm going to accept it and then will my colleague do a final review before it's finally merged. Thanks for your contribution

**Review by mattisssa — APPROVED** [review]

**Pull request merged**